### PR TITLE
[hi] Localize data-plane.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/glossary/data-plane.md` to Hindi `content/hi/docs/reference/glossary/data-plane.md`

Fixes :- #35600

### CODE REVIEW

File: content/hi/docs/reference/glossary/data-plane.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: डेटा प्लेन (Data Plane)
+id: data-plane
+date: 2019-05-12
+full_link:
+short_description: >
+  वह परत जो सीपीयू, मेमोरी, नेटवर्क और स्टोरेज जैसे क्षमता प्रदान करता है ताकि कंटेनर चल सकें और नेटवर्क से जुड़ सकें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/data-plane.md:7]:
```suggestion
  वह परत जो CPU, मेमोरी, नेटवर्क और स्टोरेज जैसी क्षमता प्रदान करता है ताकि कंटेनर चल सकें और नेटवर्क से जुड़ सकें।
```

File: content/hi/docs/reference/glossary/data-plane.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+aka:
+tags:
+  - fundamental
+---
+
+वह परत जो सीपीयू, मेमोरी, नेटवर्क और स्टोरेज जैसे क्षमता प्रदान करता है ताकि कंटेनर चल सकें और नेटवर्क से जुड़ सकें।
```
Comment (Babapool) [inline content/hi/docs/reference/glossary/data-plane.md:14]:
```suggestion
वह परत जो CPU, मेमोरी, नेटवर्क और स्टोरेज जैसी क्षमता प्रदान करता है ताकि कंटेनर चल सकें और नेटवर्क से जुड़ सकें।
```

**Review by Babapool — COMMENTED** [review]
Some small nits from myside, otherwise the PR LGTM

**Review by divya-mohan0209 — COMMENTED** [review]
/approve

**Comment by divya-mohan0209** [comment]
@Babapool : Whenever you have some cycles free, PTAL!

**Review by Babapool — COMMENTED** [review]
Thanks, the changes LGTM.
/lgtm

**Pull request merged**